daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/scenario/ScenarioLedger.scala

```diff
@@ -415,163 +415,156 @@ object ScenarioLedger {
 
     @tailrec
     def processNodes(
-        mbCache0: Either[UniqueKeyViolation, LedgerData],
+        cache0: LedgerData,
```

Comment:
for other reviewers:
we’re planning to further cleanup this loop later. But now it has close to no tricky semantic content so this is less crucial (it’s mainly  around updating referencedBy & consumedBy which are display only).

You probably also want to disable this with whitespace changes disabled.

Reply:
Ref: https://github.com/digital-asset/daml/issues/14183